docs/drafts/derive-serviceaccount.md

```diff
@@ -1,6 +1,6 @@
 # Derive minimal ServiceAccount required for ClusterExtension Installation and Management
 
-OLMv1 does not provide cluster admin privileges by default for installing cluster extensions. This means that the installation process will require a service account with sufficient privileges to install the bundle. It depends on the cluster extension developer to specify the exact permissions required for the management of any specific bundle. A ServiceAccount needs to be explicitly specified for installing and upgrading operators to avoid errors when deploying your cluster extension.
+OLMv1 does not provide cluster admin privileges by default for installing cluster extensions. This document will explain how to derive the required RBAC configuration for a provided service account"
```

Comment:
Remove quotation mark at end and replace with period

Reply:
I also think this wording could be improved. Maybe something like:

OLM v1 does not have permission to install extensions on a cluster by default. In order to install a [supported bundle](https://github.com/operator-framework/operator-controller/blob/main/docs/refs/supported-extensions.md), OLM must be provided a `ServiceAccount` configured with the appropriate permissions. For more information, see the [provided ServiceAccount](https://github.com/operator-framework/operator-controller/blob/main/docs/drafts/provided-serviceaccount.md) documentation.

This document serves as a guide for how to derive the RBAC necessary to install a bundle.